.github/workflows/build-and-push-image-test.yml

```diff
@@ -0,0 +1,82 @@
+name: Continuous delivery (Test)
+
+on:
+  push:
```

Comment:
This doesn't follow on from what you've done in conversions.
Is it possible to mimic that workflow - https://github.com/DFE-Digital/a2b-internal/blob/main/.github/workflows/build-and-push-image.yml#L8-L12
Which is manually triggered when we need to run a deployment.